### DISCUSSION TITLE
Unreal Datasmith animation run different with Itwin Datasmith Exporter

### DISCUSSION THREAD

**Discussion by dinhnha1402** [iTwin/itwinjs-core#5761] [Q&A] [answered]
Hello,
I have an error with using the animation sequence of combineMeshActor. The clipping plane runs then suddenly stop. However, the animation running in Itwin Datasmith Exporter works very well.

https://github.com/iTwin/itwinjs-core/assets/34652723/13bf0891-6a71-404a-b162-3ceb82d9d15e

https://github.com/iTwin/itwinjs-core/assets/34652723/65c635d2-deb3-4f12-850d-f216c8462457

**Comment by pmconne** [comment]
@MichaelBelousov

**Comment by MichaelBelousov** [comment, marked as answer]
Hi @dinhnha1402 

We launched the iTwin Exporter for Datasmith last year as a Technology Preview; we received great feedback from early adopters. One of the most significant drawbacks of the iTwin Exporter for Datasmith that we received feedback about was that the file format is unsuitable for large-scale infrastructure projects. One of the core tenets of the iTwin Platform is scalable support for large-scale modern infrastructure projects. With this limitation identified, we have developed the 3DFT technology, which supports large-scale infrastructure model streaming in Unreal Engine. 3DFT is replacing the iTwin Exporter for Datasmith, and this is a bug in the iTwin Exporter for Datasmith.

To get started with 3DFT, register an application on the iTwin Platform by following [these steps](https://developer.bentley.com/tutorials/quickstart-web-and-service-apps/#12-register-your-application).

You can get more information about how to load your iTwin in Unreal at our [Unreal plugin's page](https://github.com/iTwin/unreal-engine-3dft-plugin)
